### PR TITLE
chore: convert locale to typescript

### PR DESCRIPTION
**Pull request by suevalov**

### CODE REVIEW

File: typings/generated/entities/locale.d.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import { AxiosInstance } from 'axios';
+import { Except, SetOptional } from 'type-fest';
```
Comment (marcolink) [inline typings/generated/entities/locale.d.ts:2]: I'm just curious, what is the difference between `setOptional` and the builtin `Partial`?
Same for `Except`  and `Exclude`

Comment (marcolink) [reply typings/generated/entities/locale.d.ts:2]: Nevermind, looked at the docs 👍

Comment (suevalov) [reply typings/generated/entities/locale.d.ts:2]: The `Omit` type does not restrict the omitted keys to be keys present on the given type, while `Except` does.

`Partial` marks all properties optional while `SetOptional` only those that mentioned keeping other untouched.

**Pull request merged**